interface/frontend/src/components/detect-and-select/CandidateList.vue

```diff
@@ -1,46 +1,157 @@
+                    <a @click="dismiss(candidate)">
+                      <button type="button" class="btn btn-sm btn-secondary">Dismiss</button>
+                    </a>
+                    <a @click="mark(candidate)">
+                      <button type="button" class="btn btn-sm btn-danger">Mark concerning</button>
+                    </a>
+                  </div>
+                </div>
+              </div>
+
+            </template>
+          </div>
+        </div>
+        <div class="col-md-8">
+          <open-dicom :view="viewerData" :marker="marker"></open-dicom>
+        </div>
       </div>
-      </template>
-      <template v-else>
-        <p class="card-text">No candidates available.</p>
-      </template>
-    </div>
-    <div class="col-md-8"></div>
+    </template>
+    <template v-else>
+      <p class="card-text">No candidates available.</p>
+    </template>
+    <button @click="showDicom">Show DICOM</button>
   </div>
-</div>
 </template>
 
 <script>
-import Candidate from './Candidate'
+  import {EventBus} from '../../main.js'
+  import Candidate from './Candidate'
+  import OpenDicom from '../open-image/OpenDICOM'
 
-export default {
-  components: { Candidate },
-  data () {
-    return {
-      candidates: []
-    }
-  },
-  created () {
-    this.fetchCandidates()
-  },
-  methods: {
-    fetchCandidates () {
-      this.$axios.get('/api/candidates/')
-        .then((response) => {
-          this.candidates = response.data
-        })
-        .catch(() => {
-          // TODO: error callback
-        })
+  export default {
+    components: {Candidate, OpenDicom},
+    data () {
+      return {
+        candidates: [],
+        selectedCandidateIndex: -1,
+        viewerData: {
+          type: 'DICOM',
+          prefixCS: ':/',
+          prefixUrl: '/api/images/metadata?dicom_location=/',
+          paths: []
+        }
+      }
+    },
+    computed: {
+      marker () {
+        if (!this.selectedCandidate) {
+          return null
+        }
+
+        var noduleCentroid = this.selectedCandidate.centroid
+        return noduleCentroid ? {x: noduleCentroid.x, y: noduleCentroid.y, z: noduleCentroid.z} : null
+      },
+      selectedCandidate () {
+        return this.candidates[this.selectedCandidateIndex] || null
+      }
+    },
+    created () {
+      this.fetchCandidates()
+    },
+    mounted: function () {
+      EventBus.$on('nodule-marker-moved', (x, y, z) => {
+        if (!this.selectedCandidate) {
+          console.error('can\'t save new coordinates - selectedCanidate is not found')
+          return
+        }
+
+        const selectedCandidate = this.selectedCandidate
+
+        // mark current candidate as saving
+        selectedCandidate._saving = true
+
+        // send data to backend
+        this.$axios
+            .post(selectedCandidate.url + 'move', {
+              x: x,
+              y: y,
+              z: z
```

Comment:
This can be simplified with ES6:
```
 .post(selectedCandidate.url + 'move', {x, y, z }
```
JS will grab the variable by name and apply its value as a key-value pair by itself :p